tm2/pkg/sdk/auth/consts.go

```diff
@@ -11,8 +11,8 @@ const (
 	// StoreKey is string representation of the store key for auth
 	StoreKey = "acc"
 
-	// FeeCollectorName the root string for the fee collector account address
-	FeeCollectorName = "fee_collector"
+	// FeeCollectorKey the params key for the fee collector account address
+	FeeCollectorKey = "p:fee_collector"
```

Comment:
We have access to `DefaultFeeCollectorName`

Reply:
The more I look at this constant (value, with the `p`), the more I think we should probably move it to `params.go`

Check out the VM params and how we define paths there to be consistent